### PR TITLE
Radial (suburst) chart for pool space widget

### PR DESCRIPTION
**Pull request by surajrav**
This is in no way even working (no pure d3 functionality even there as yet).

I shall ping appropriate people when I have finished this PR and it is ready to be merged.

This is just so that they (mostly @pchaussalet) can comment on things and provide me with some feedback as I go.

Ticket: #8246

### CODE REVIEW

File: ui/controls/radial-chart.reel/_radial-chart.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.RadialChart {
```
Comment (thibaultzanini) [inline ui/controls/radial-chart.reel/_radial-chart.css:1]: @joshuarule Maybe you should review the css here.

Comment (surajrav) [reply ui/controls/radial-chart.reel/_radial-chart.css:1]: @thibaultzanini this is ready for a checkpoint PR to be done.

Spoke to @pchaussalet and he agreed that I should at least have this much merged in for now.

**Comment by thibaultzanini** [comment]
@surajrav This branch as conflict with master could you fix it please thanks.

**Comment by pchaussalet** [comment]
@surajrav @jordanhubbard I think this is obsolete as it will be replaced by single dataset pie chart, right ?

**Comment by surajrav** [comment]
I dunno about the replacement as a single disk pie chart or not but yes this is obsolete for sure, also the ticket tracking this just got pushed to 10-SU1 so closing this for now

**Pull request closed without merging**